src/libraries/System.Private.Xml/src/System/Xml/Serialization/CodeIdentifier.cs

```diff
@@ -283,7 +283,7 @@ private static void EscapeKeywords(string identifier, StringBuilder sb)
             {
                 if (separator >= 0)
                 {
-                    sb.Append(originalIdentifier.Substring(separator, 1));
+                    sb.Append(originalIdentifier.AsSpan(separator, 1));
```

Comment:
With a count of 1, would this not be better as:
```suggestion
                    sb.Append(originalIdentifier[separator]);
```
?

Reply:
Good catch.